hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/clustering/HoodieFlinkClusteringJob.java

```diff
@@ -348,7 +348,8 @@ private void cluster() throws Exception {
           .addSink(new ClusteringCommitSink(conf))
           .name("clustering_commit")
           .uid("uid_clustering_commit")
-          .setParallelism(1);
+          .setParallelism(1)
+          .setMaxParallelism(1);
```

Comment:
Nice catch, just need to fix the compatibility issues: https://github.com/apache/hudi/actions/runs/6864369897/job/18665977391?pr=10090

Reply:
Seems that only Flink 1.16+ has this `setParallelism` API ?

Reply:
> Seems that only Flink 1.16+ has this `setParallelism` API ?

Maybe 1.18+